bin/lux

```diff
@@ -32,6 +33,14 @@ function inLuxProject() {
   }
 }
 
+function commandNotFound(cmd) {
+  console.log();
```

Comment:
These empty `console.log`'s are a little hard to understand. I'm assuming this is to add a line break? If so, I suggest we do the following (It's not any more pretty but it's a bit more obvious what's going on imo):

```javascript
const { EOL } = require('os');

function commandNotFound(cmd) {
  console.log(`${EOL}  ${red(cmd)} isn't a valid command${EOL}`);
  console.log(`  Type ${green('lux --help')} for a full list of commands${EOL}`);
}
```

--

[`os.EOL`](https://nodejs.org/dist/latest-v6.x/docs/api/os.html#os_os_eol) is used to determine the appropriate line breaks for platform Lux is installed on.